nixos/tests/lxqt.nix

```
@@ -37,7 +37,7 @@
       with subtest("Wait for login"):
           machine.wait_for_x()
           machine.wait_for_file("/tmp/xauth_*")
-          machine.sleep(1)
+          machine.wait_until_succeeds("test -s /tmp/xauth_*")
```

Why not `wait_for_file` ?
https://github.com/NixOS/nixpkgs/blob/baa3367ea6e7f220e8fdda07445878d9a33525cd/nixos/lib/test-driver/src/test_driver/machine/__init__.py#L727

`test -e` does work with globs. I suppose it doesn't check whether it is a socket, but does that matter that much if we didn't care before about whether it was sockets? THis feels a bit like reinventing the wheel...

```suggestion
          machine.wait_for_file("/tmp/xauth_*")
```